### DISCUSSION TITLE
[Idea] Introduce alternative search panels same page links

### DISCUSSION THREAD

**Discussion by sowbiba** [PrestaShop/PrestaShop#33744] [Ideas]
### The problem

Alternative search panels (external resources when making a global search in the BO) links always have target="_blank" but what if we want to have internal page link?

### The solution

Add a property to allow opening the link in the same page

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by sowbiba** [comment]
Check https://github.com/PrestaShop/PrestaShop/pull/33745

**Comment by kpodemski** [comment]
PR has been merged 🎉
